app/helpers/record-meta.js

```diff
@@ -0,0 +1 @@
+export { default, recordMeta } from 'ember-data-record-meta/helpers/record-meta'
```

Comment:
I think this isn't up-to-date anymore? I don't see a `recordMeta` export?

Reply:
Correct, I'll remove that.

Reply:
Removed.